namespace-specific-policy/configsync-src/external-team/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# deny all ingress traffic to avoid Pods from one namespace
+# accidentally sending traffic to Services in other namespaces.
+# [START anthosconfig_namespace_specific_policy_src_external_team_kustomization] 
+namespace: external-team
+
+resources:
+- namespace.yaml
+- rolebinding.yaml
+- role.yaml
+- networkpolicy.yaml
```

Comment:
Instead of having another copy of these resources, can the external-team reference the base kustomization.yaml file?

Reply:
Done